src/Components/WebAssembly/WebAssembly/src/Services/WebAssemblyDynamicResourceLoader.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using System.Runtime.Loader;
+using Microsoft.JSInterop.WebAssembly;
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Services
+{
+    public class WebAssemblyDynamicResourceLoader
```

Comment:
Should this have the word assembly in it's name?

Reply:
Good call.